static/app/views/alerts/rules/metric/details/metricChartOption.tsx

```diff
@@ -355,6 +403,72 @@ export function getMetricAlertChartOption({
       });
   }
 
+  if (anomalies) {
+    const anomalyBlocks: any[] = [];
+    let start;
+    let end;
+    anomalies
+      .filter(anomalyts => {
+        const ts = new Date(anomalyts.timestamp).getTime();
+        return firstPoint < ts && ts < lastPoint;
+      })
+      .forEach(anomalyts => {
+        const {anomaly, timestamp} = anomalyts;
+
+        if (
+          [AnomalyType.high, AnomalyType.low].indexOf(anomaly.anomaly_type as string) > -1
+        ) {
+          if (!start) {
+            start = new Date(timestamp).toISOString();
+          }
```

Comment:
a comment here might help the next person, like what does it mean to not have the start here